.circleci/config.yml

```diff
@@ -1896,7 +1896,6 @@ workflows:
                 - alpine:3.10
                 - alpine:3.11
                 - alpine:3.12
-                - alpine:latest
```

Comment:
Why remove? Don't we need to support it?

Reply:
Ah, yeah, I only updated the commit message, but not the PR desc: latest alpine has no php7 package anymore.

Reply:
Ah, I see. Maybe add 3.15 which is the last one to have it?